vertx-config/src/main/java/io/vertx/config/ConfigStoreOptions.java

```diff
@@ -0,0 +1,78 @@
+package io.vertx.config;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+
+import java.util.Objects;
+
+/**
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+@DataObject
+public class ConfigStoreOptions {
+
+  private String type;
+
+  private JsonObject config;
+  private String format;
+
+  public ConfigStoreOptions() {
+    // Empty constructor
+  }
+
+  public ConfigStoreOptions(ConfigStoreOptions other) {
+    this.type = other.type;
+    this.config = other.config;
+  }
+
+  public ConfigStoreOptions(JsonObject json) {
+    type = json.getString("type");
+    config = json.getJsonObject("config", new JsonObject());
```

Comment:
why do we get an empty json object `config` if no one is present as in the constructor the `config` we have null ?